pkg/runtest/run_test.go

```diff
@@ -16,6 +17,11 @@ import (
 	_ "github.com/google/syzkaller/sys/test/gen" // pull in the test target
 )
 
+// Can be used as:
+// go test -v -run=Test/64_fork ./pkg/runtest -filter=nonfailing
+// to select a subset of tests to run.
+var flagFilter = flag.String("filter", "", "prefix to match test file names")
```

Comment:
We already have `flagTests` in `tools/syz-runtest/runtest.go`, this seems to be the same thing

Reply:
Yes, this flag does the same, but it's for `go test ./pkg/runtest`. And the other one if for tools/syz-runtest.